x-pack/platform/plugins/private/upgrade_assistant/server/lib/reindexing/index_settings.ts

```diff
@@ -125,63 +113,3 @@ export const getReindexWarnings = (flatSettings: FlatSettings): ReindexWarning[]
 
   return warnings;
```

Comment:
Not necessarily related to this PR, but I thought the above "deprecated setting detection" logic could also be avoided. Perhaps it's a holdover from 7 🤷🏻 ?

Reply:
good point! I'll remove it as well.